studies/templates/studies/study_responses.html

```diff
@@ -8,11 +8,10 @@
     <script type="application/javascript">
         $(document).ready(function(){
             hideResponses();
-            prepareDataForDownload('.all-data', 'download-all-data', 'json');
             prepareDataForDownload('#json-data-1', 'download-individual-data', 'json');
-            $(".all-csv-data").hide();
             $( ".selectable-participant" ).first().addClass('selected');
             $( ".response-data" ).first().show();
+            $( ".response-attachments" ).first().show();
```

Comment:
`$(".response-attachments")` why the extra spaces ? Some weird editor formatting?